src/lib/units/offset.js

```diff
@@ -161,11 +161,10 @@ export function setOffsetToLocal (keepLocalTime) {
 }
 
 export function setOffsetToParsedOffset () {
-    if (this._tzm) {
+    if (this._tzm === 0 || this._tzm) {
```

Comment:
Can we write this in a more readable fashion? Like is it a number of a non-empty string.